<launch>
  <arg name="cmd_vel" default="marker_vel"/>

  <node pkg="marker_teleop" type="twist_marker_teleop_node" name="twist_marker_teleop"
        output="screen">
    <remap from="cmd_vel" to="$(arg cmd_vel)"/>

    <rosparam file="$(find marker_teleop)/config/twist_marker_teleop.yaml" command="load"/>
  </node>
</launch>
